### ISSUE TITLE
Multiline strings aren't converted correctly

### ISSUE THREAD

**Issue by alexgorbatchev** [eventualbuddha/decaffeinate#69]
```coffeescript
describe 'regular brush', ->
  before ->
    render """
        hello world
        how are things?
    """
```

gets translated to

```javascript
describe('regular brush', function() {
  return before(function() {
    return render("""
        hello world
        how are things?
    """);
  });
});
```

**Comment by eventualbuddha** [comment]
Fixed in v1.40.9.

**Comment by eventualbuddha** [comment]
Oops, this is incorrect. It appears CoffeeScript eats all leading whitespace in triple-quoted strings:

```js
describe('regular brush', function() {
  return before(function() {
    return render("hello world\nhow are things?");
  });
});
```

**Comment by eventualbuddha** [comment]
Fixed (again) in v1.40.11.

**Comment by eventualbuddha** [comment]
Also, this should apply just as much to interpolated strings:

```coffee
describe 'regular brush', ->
  before ->
    render """
        hello #{name}
        how are things?
    """
```

```js
describe('regular brush', function() {
  return before(function() {
    return render(`hello ${name}
how are things?`);
  });
});
```

**Comment by eventualbuddha** [comment]
Third time's the charm, I hope. Fixed in v1.40.12:

```js
describe('regular brush', function() {
  return before(function() {
    return render(`hello world
how are things?`);
  });
});
```

**Comment by eventualbuddha** [comment]
One more fix for triple-quoted strings with interpolation released in v1.40.13.
